explainer.md

```diff
@@ -86,39 +116,73 @@ Note that as part of this proposal, we are proposing to [remove the requirement]
 
 ### Acknowledging a purchase
 
-Some stores will require that the user acknowledge a purchase once it has succeeded. In this case, the payment response will return a "purchase token" string, which can be used with the `acknowledge` method.
-
-Items that are designed to be purchased multiple times must be acknowledged with the `repeatable` flag. An example of a repeatable purchase is an in-game powerup that makes the player stronger for a short period of time. Once it is acknowledged with the `repeatable` flag, it can be purchased again.
-
-```js
-digitalGoodsService.acknowledge(purchaseToken, 'repeatable');
-```
+Some stores will require that the developer acknowledge a purchase once it has succeeded. In this case, the payment response will return a "purchase token" string, which should be verified and acknowledged by the developer. This should be done using direct API calls from the developer's server to the service provider before granting entitlements. Such a direct developer-to-provider API is not specified as part of the Digital Goods API.
```

Comment:
Let's make this more open ended given the flow here is not supported by the API. Something like: `Payment response returns a "purchase token" string. This can be used for direct communication with the provider beyond the DGAPI e.g. a client side ack that the purchase succeeded.`

Reply:
True. Done.